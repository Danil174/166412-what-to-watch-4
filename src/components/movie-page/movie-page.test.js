import React from "react";
import renderer from "react-test-renderer";
import MoviePage from "./movie-page.jsx";

describe(`MoviePageTest`, () => {
  it(`render MoviePage`, () => {
    const tree = renderer
      .create(
          <MoviePage />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
