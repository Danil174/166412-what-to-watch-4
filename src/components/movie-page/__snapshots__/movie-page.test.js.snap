// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePageTest render MoviePage 1`] = `
Array [
  <section
    className="movie-card movie-card--full"
    style={
      Object {
        "backgroundColor": "#A6B7AC",
      }
    }
  >
    <div
      className="movie-card__hero"
    >
      <div
        className="movie-card__bg"
      >
        <img
          alt="Gangs of new york"
          src="https://htmlacademy-react-3.appspot.com/wtw/static/film/background/gangs_of_new_york.jpg"
        />
      </div>
      <h1
        className="visually-hidden"
      >
        WTW
      </h1>
      <header
        className="page-header movie-card__head"
      >
        <div
          className="logo"
        >
          <a
            className="logo__link"
            href="/"
          >
            <span
              className="logo__letter logo__letter--1"
            >
              W
            </span>
            <span
              className="logo__letter logo__letter--2"
            >
              T
            </span>
            <span
              className="logo__letter logo__letter--3"
            >
              W
            </span>
          </a>
        </div>
        <div
          className="user-block"
        >
          <a
            className="user-block__link"
            href="/login"
          >
            Sign in
          </a>
        </div>
      </header>
      <div
        className="movie-card__wrap"
      >
        <div
          className="movie-card__desc"
        >
          <h2
            className="movie-card__title"
          >
            Gangs of new york
          </h2>
          <p
            className="movie-card__meta"
          >
            <span
              className="movie-card__genre"
            >
              Crime
            </span>
            <span
              className="movie-card__year"
            >
              2002
            </span>
          </p>
          <div
            className="movie-card__buttons"
          >
            <button
              className="btn btn--play movie-card__button"
              type="button"
            >
              <svg
                height="19"
                viewBox="0 0 19 19"
                width="19"
              >
                <use
                  xlinkHref="#play-s"
                />
              </svg>
              <span>
                Play
              </span>
            </button>
            <button
              className="btn btn--list movie-card__button"
              onClick={[Function]}
              type="button"
            >
              <svg
                height="20"
                viewBox="0 0 19 20"
                width="19"
              >
                <use
                  xlinkHref="#add"
                />
              </svg>
              <span>
                My list
              </span>
            </button>
            <a
              className="btn movie-card__button"
              href="add-review.html"
            >
              Add review
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="movie-card__wrap movie-card__translate-top"
    >
      <div
        className="movie-card__info"
      >
        <div
          className="movie-card__poster movie-card__poster--big"
        >
          <img
            alt="Gangs of new york poster"
            height="327"
            src="https://htmlacademy-react-3.appspot.com/wtw/static/film/poster/Gangs_of_New_York_Poster.jpg"
            width="218"
          />
        </div>
        <div
          className="movie-card__desc"
        >
          <nav
            className="movie-nav movie-card__nav"
            onClick={[Function]}
          >
            <ul
              className="movie-nav__list"
            >
              <li
                className="movie-nav__item movie-nav__item--active"
              >
                <a
                  className="movie-nav__link"
                  data-item="Overview"
                >
                  Overview
                </a>
              </li>
              <li
                className="movie-nav__item 0"
              >
                <a
                  className="movie-nav__link"
                  data-item="Details"
                >
                  Details
                </a>
              </li>
              <li
                className="movie-nav__item 0"
              >
                <a
                  className="movie-nav__link"
                  data-item="Reviews"
                >
                  Reviews
                </a>
              </li>
            </ul>
          </nav>
          <div
            className="movie-rating"
          >
            <div
              className="movie-rating__score"
            >
              10
            </div>
            <p
              className="movie-rating__meta"
            >
              <span
                className="movie-rating__level"
              >
                Awesome
              </span>
              <span
                className="movie-rating__count"
              >
                370881
                 ratings
              </span>
            </p>
          </div>
          <div
            className="movie-card__text"
          >
            In 1862, his father's killer.
            <p
              className="movie-card__director"
            >
              <strong>
                Director: 
                Martin Scorsese
              </strong>
            </p>
            <p
              className="movie-card__starring"
            >
              <strong>
                Starring: 
                Leonardo DiCaprio, Cameron Diaz, Daniel Day-Lewis
                 and other
              </strong>
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <div
    className="page-content"
  >
    <section
      className="catalog catalog--like-this"
    >
      <h2
        className="catalog__title"
      >
        More like this
      </h2>
      <div
        className="catalog__movies-list"
      >
        <article
          className="small-movie-card catalog__movies-card"
          onMouseOut={[Function]}
          onMouseOver={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="100%"
              width="100%"
            />
          </div>
          <h3
            className="small-movie-card__title"
            onClick={[Function]}
          >
            <a
              className="small-movie-card__link"
              href="movie-page.html"
            >
              Gangs of new york
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseOut={[Function]}
          onMouseOver={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="100%"
              width="100%"
            />
          </div>
          <h3
            className="small-movie-card__title"
            onClick={[Function]}
          >
            <a
              className="small-movie-card__link"
              href="movie-page.html"
            >
              Gangs of new york
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseOut={[Function]}
          onMouseOver={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="100%"
              width="100%"
            />
          </div>
          <h3
            className="small-movie-card__title"
            onClick={[Function]}
          >
            <a
              className="small-movie-card__link"
              href="movie-page.html"
            >
              Gangs of new york
            </a>
          </h3>
        </article>
      </div>
    </section>
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link logo__link--light"
          href="/"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="copyright"
      >
        <p>
          Â© 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>,
]
`;
